src/element_style.py

```
@@ -0,0 +1,533 @@
+                continue
+            properties[element_key] = _style_element_block(element_key, declaration)
+            if isinstance(order, list) and element_key not in order:
+                # Keep generated elements ahead of the layout section.
+                insert_at = order.index("layout") if "layout" in order else len(order)
+                order.insert(insert_at, element_key)
+
+        if offset_elements:
+            _expand_offset_blocks(properties, order, offset_elements)
+
+        return schema
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)